training/tf/tfprocess.py

```diff
@@ -534,7 +527,7 @@ def add_weights(self, var):
             assert var.dtype.base_dtype == tf.float32
             self.weights.append(var)
 
-    def batch_norm(self, net):
+    def batch_norm(self, net, scale=True):
```

Comment:
Yay for finally fixing this. But there's a format description in the README.md, so we'll have to decide what to do with that.